test/cmd/builds.sh

```diff
@@ -13,55 +14,55 @@ project="$(oc project -q)"
 
 # This test validates builds and build related commands
 
-oc new-build openshift/ruby-20-centos7 https://github.com/openshift/ruby-hello-world.git
-oc get bc/ruby-hello-world
-cat "${OS_ROOT}/Dockerfile" | oc new-build -D - --name=test
-oc get bc/test
-oc new-build --dockerfile=$'FROM centos:7\nRUN yum install -y httpd'
-oc get bc/centos
-oc delete all --all
+os::cmd::expect_success 'oc new-build openshift/ruby-20-centos7 https://github.com/openshift/ruby-hello-world.git'
+os::cmd::expect_success 'oc get bc/ruby-hello-world'
+os::cmd::expect_success 'cat "${OS_ROOT}/Dockerfile" | oc new-build -D - --name=test'
+os::cmd::expect_success 'oc get bc/test'
+os::cmd::expect_success "oc new-build --dockerfile=\$'FROM centos:7\nRUN yum install -y httpd'"
+os::cmd::expect_success 'oc get bc/centos'
+os::cmd::expect_success 'oc delete all --all'
 
-oc process -f examples/sample-app/application-template-dockerbuild.json -l build=docker | oc create -f -
-oc get buildConfigs
-oc get bc
-oc get builds
+os::cmd::expect_success 'oc process -f examples/sample-app/application-template-dockerbuild.json -l build=docker | oc create -f -'
+os::cmd::expect_success 'oc get buildConfigs'
+os::cmd::expect_success 'oc get bc'
+os::cmd::expect_success 'oc get builds'
 
 # make sure the imagestream has the latest tag before starting a build or the build will immediately fail.
-wait_for_command "oc get is ruby-20-centos7 | grep latest" $((60*TIME_SEC))
+wait_for_command "os::cmd::expect_success_and_text 'oc get is ruby-20-centos7' 'latest'" $((60*TIME_SEC))
 
 REAL_OUTPUT_TO=$(oc get bc/ruby-sample-build --template='{{ .spec.output.to.name }}')
-oc patch bc/ruby-sample-build -p '{"spec":{"output":{"to":{"name":"different:tag1"}}}}'
-oc get bc/ruby-sample-build --template='{{ .spec.output.to.name }}' | grep 'different'
-oc patch bc/ruby-sample-build -p "{\"spec\":{\"output\":{\"to\":{\"name\":\"${REAL_OUTPUT_TO}\"}}}}"
+os::cmd::expect_success "oc patch bc/ruby-sample-build -p '{\"spec\":{\"output\":{\"to\":{\"name\":\"different:tag1\"}}}}'"
+os::cmd::expect_success "oc get bc/ruby-sample-build --template='{{ .spec.output.to.name }}' | grep 'different'"
+os::cmd::expect_success "oc patch bc/ruby-sample-build -p '{\"spec\":{\"output\":{\"to\":{\"name\":\"${REAL_OUTPUT_TO}\"}}}}'"
 echo "patchAnonFields: ok"
 
-[ "$(oc describe buildConfigs ruby-sample-build | grep --text "Webhook GitHub" | grep -F "${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/github")" ]
-[ "$(oc describe buildConfigs ruby-sample-build | grep --text "Webhook Generic" | grep -F "${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic")" ]
```

Comment:
I don't really understand what `grep -F` is doing here, as there are no newline delimiters. I got rid of it.